config.json

```diff
@@ -583,8 +583,7 @@
         "name": "Darts",
         "uuid": "4d400a44-b190-4a0c-affb-99fad8ea18da",
         "practices": [
-          "if-else-statements",
-          "constructors"
+          "if-else-statements"
         ],
         "prerequisites": [
```

Comment:
I see that the "constructors" concept is also in the list of prerequisites for this exercise. This means that the exercise is only unlocked after completing the learning exercise covering the "constructors" concept.

Since we agreed that the concept is not applicable for this exercise, we should also remove it as a prerequisite by removing it from this list.

Reply:
@mic-en I see you resolved this, did you make the change but forget to push perhaps? I don't see any new commits on this PR.